From 77612d9fafa0604447bae0c36781ed91f9c1c0ca Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 26 Aug 2014 12:49:34 -0400
Subject: [tools] cpupower: Rename cpufrequtils -> cpupower, and libcpufreq -> libcpupower

Message-id: <1409057382-26432-24-git-send-email-prarit@redhat.com>
Patchwork-id: 89094
O-Subject: [RHEL7.1 PATCH BZ 1133893 23/31] cpupower: Rename cpufrequtils -> cpupower, and libcpufreq -> libcpupower.
Bugzilla: 1133893
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1133893

commit a504c028c96b738d1579b0bfe73782f80d8696f6
Author: Ramkumar Ramachandra <artagnon@gmail.com>
Date:   Tue May 13 12:41:38 2014 +0200

    cpupower: Rename cpufrequtils -> cpupower, and libcpufreq -> libcpupower.

    Signed-off-by: Ramkumar Ramachandra <artagnon@gmail.com>
    Signed-off-by: Thomas Renninger <trenn@suse.de>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/power/cpupower/README b/tools/power/cpupower/README
index fd9d4c0..96ff1ff 100644
--- a/tools/power/cpupower/README
+++ b/tools/power/cpupower/README
@@ -1,4 +1,4 @@
-The cpufrequtils package (homepage: 
+The cpupower package (homepage:
 http://www.kernel.org/pub/linux/utils/kernel/cpufreq/cpufrequtils.html ) 
 consists of the following elements:
 
@@ -11,10 +11,10 @@ providing cpuid.h is needed.
 For both it's not explicitly checked for (yet).
 
 
-libcpufreq
+libcpupower
 ----------
 
-"libcpufreq" is a library which offers a unified access method for userspace
+"libcpupower" is a library which offers a unified access method for userspace
 tools and programs to the cpufreq core and drivers in the Linux kernel. This
 allows for code reduction in userspace tools, a clean implementation of
 the interaction to the cpufreq core, and support for both the sysfs and proc
@@ -28,7 +28,7 @@ make
 su
 make install
 
-should suffice on most systems. It builds default libcpufreq,
+should suffice on most systems. It builds default libcpupower,
 cpufreq-set and cpufreq-info files and installs them in /usr/lib and
 /usr/bin, respectively. If you want to set up the paths differently and/or
 want to configure the package to your specific needs, you need to open
@@ -39,11 +39,11 @@ CONFIGURATION.
 THANKS
 ------
 Many thanks to Mattia Dongili who wrote the autotoolization and
-libtoolization, the manpages and the italian language file for cpufrequtils;
+libtoolization, the manpages and the italian language file for cpupower;
 to Dave Jones for his feedback and his dump_psb tool; to Bruno Ducrot for his
 powernow-k8-decode and intel_gsic tools as well as the french language file;
 and to various others commenting on the previous (pre-)releases of 
-cpufrequtils.
+cpupower.
 
 
         Dominik Brodowski
-- 
1.7.1